ss_admin_console/main/default/classes/SSACController.cls

```diff
@@ -54,4 +54,16 @@ public with sharing class SSACController {
         item.execute();
         return 'Execution Started';
     }
+
+    @AuraEnabled
+    public static Boolean getProductionOrgToggle(){
```

Comment:
There is no need to have two methods getProductionOrgToggle() and showActionDropdown().

It's enough to have one of them